src/schema/v2/searchCriteriaLabel.ts

```diff
@@ -117,7 +124,8 @@ async function getArtistLabels(artistIDs: string[], artistLoader) {
       const artist = await artistLoader(id)
       return {
         name: "Artist",
-        value: artist.name,
```

Comment:
**question**: This might be a disruptive change as we already have frontend clients (including mobile apps) using this `value` attribute for the pill label. Would it be less disruptive to add a new `rawValue` attribute and leave the existing `value` attribute as is?

I like the naming in this PR but we might be able to jump to it right away.

Reply:
Eigen doesn't use these changes. We will need to make changes only for Force. And I wanted to consult with you about this later and clarify how to do better for Force:
* temporarily comment out [changes](https://github.com/artsy/force/pull/9526) –> push commented changes -> wait for staging/production build - make the necessary changes according to the updated scheme and push it
* make new changes according to the updated scheme and push it, but in such a situation, when opening on `user/alerts`, there will be a 500 error (temporarily, until new changes are merged)
* use a different naming `rawValue` (as suggested by you)

Reply:
> Eigen doesn't use these changes

Ah, I didn't realize we were only using these labels on Force. That simplifies things a bit.

Although, looking at the full query, would it be odd to have label as an attribute on the SearchCriteriaLabel type? labels  > label? Maybe `rawValue` is a bit more intention revealing?

cc/ @anandaroop if you have thoughts on the naming

```graphql
{
  me {
    savedSearchesConnection(first: 5) {
      edges {
	node {
	  labels {
	    name
	    field
	    value
	  }
	}
      }
    }
  }
}

```

Reply:
Interesting, I agree labels > label fields a little weird.

To summarize, for my own benefit...

```gql
# 1) currently live

labels {
  name      # sizes
  field     # Size
  value     # Small (under 40cm)"
  <missing> # SMALL
 }

# 2) this PR

labels {
  name   # sizes
  field  # Size
  value  # SMALL
  label  # Small (under 40cm)"
 }

# 3) devon suggestion

labels {
  name     # sizes
  field    # Size
  rawValue # SMALL
  value    # Small (under 40cm)"
 }

# 4) another option?

labels {
  name         # sizes
  field        # Size
  value        # SMALL
  displayValue # Small (under 40cm)"
 }
```

I'd be fine with 3 or 4 — and if backwards compatibility is a concern, that seems to point to 3.

Works for me!

Reply:
I know we use `displayValue` elsewhere so that's a good suggestion.

@dimatretyak Up to you! We can go with #4 if it's not too much trouble to temporary disable the feature in Force. Otherwise, #3 would also work but I don't love the `rawValue` attribute name.

Reply:
I agree with 4 👍

Reply:
@dblandin @anandaroop thanks for the discussion and help!